submissions/vakulinina/html-movie-seat-booking/style.css

```diff
@@ -0,0 +1,319 @@
+* {
+  box-sizing: border-box;
+}
+
+body {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: 100vh;
+  margin: 0;
+}
+
+main {
+  display: flex;
+  flex-flow: column wrap;
+  min-width: 400px;
+  width: 1000px;
+  height: 800px;
+  border-radius: 30px;
+  background-color: #151933;
+  font-family: 'Montserrat', sans-serif;
+  box-shadow: 0 10px 20px 5px gray;
+  overflow: hidden;
+}
+
+header {
+  width: 60%;
+  padding: 10px 30px 0 30px;
+  border-bottom: 2px solid #272a43;
+}
+
+h1 {
+  color: white;
+  font-weight: 600;
+  white-space: nowrap;
+}
+
+h2 {
+  color: white;
+  font-weight: 600;
+}
+
+p {
+  color: #99a3bc;
+  font-weight: 200;
+}
+
+form {
+  width: 60%;
+  height: 300px;
+}
+
+.movie {
+  position: relative;
+  order: -1;
+  width: 40%;
+  height: 100%;
+  border-right: 2px solid #272a43;
+}
+
+.movie-poster {
+  position: absolute;
+  top: 0;
+  width: 100%;
+  height: 70%;
+  background: url(img/movie_poster.jpg) center no-repeat;
+  background-size: cover;
+}
+
+.movie-description {
+  position: absolute;
+  bottom: 0;
+  height: 33%;
+  padding: 40px 20px 0;
+  margin-bottom: 30px;
+  background: linear-gradient(180deg, rgba(20,25,51,0) 0%, rgba(20,25,51,0.75) 8%, rgba(20,25,51,0.88) 11%, rgba(20,25,51,1) 20%);
+  overflow-y: auto;
+}
+
+fieldset {
+  margin-top: 20px;
+  border: none;
+  color: #99a3bc;
+}
+
+legend {
+  padding-left: 20px;
+}
+
+input {
+  -webkit-appearance: none;
+  -moz-appearance: none;
+  appearance: none;
+}
+
+.date-selector {
+  display: flex;
+  justify-content: space-around;
+  width: 100px;
+}
+
+.date-label {
```

Comment:
Add styles on `:hover`, `:active`, and `:focus` states.
Also, add `cursor: pointer` for better identification of interactive elements.

Reply:
I already have styles for `:focus` state on the **inputs**. Will add the styles for the rest of the states.